ruff: 0.0.272 -> 0.0.273; python310Packages.ruff-lsp: 0.0.32 -> 0.0.34

figsoda opened the pull request:
Diff: https://github.com/astral-sh/ruff/compare/v0.0.272...v0.0.273

Changelog: https://github.com/astral-sh/ruff/releases/tag/v0.0.273

###### Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

figsoda committed:
ruff: 0.0.272 -> 0.0.273

Diff: https://github.com/astral-sh/ruff/compare/v0.0.272...v0.0.273

Changelog: https://github.com/astral-sh/ruff/releases/tag/v0.0.273

SuperSandro2000 commented on pkgs/development/python-modules/ruff-lsp/default.nix:
```diff
@@ -15,16 +16,25 @@
   };
 
+  patches = [
+    # fix tests with ruff >= 0.0.273
```
```suggestion
    # fix tests with ruff >= 0.0.273
    # https://github.com/astral-sh/ruff-lsp/pull/164
```

figsoda commented:
ruff-lsp 0.0.34 is coming soon which includes the patch, we can wait for that instead

figsoda committed:
python310Packages.ruff-lsp: 0.0.32 -> 0.0.34

also adds figsoda as a maintainer

Changelog: https://github.com/astral-sh/ruff-lsp/releases/tag/v0.0.34
